### PR TITLE
feat: add command to get aggregate report

### PR DESCRIPTION
**Pull request by infinitewarp**
Relates to JIRA: DISCOVERY-611

This PR adds a new `aggregate` subcommand under `report`.

```
❯ poetry run qpc report aggregate --help
usage: qpc report aggregate [-h] (--scan-job SCAN_JOB_ID | --report REPORT_ID)

options:
  -h, --help            show this help message and exit
  --scan-job SCAN_JOB_ID
                        Scan job identifier.
  --report REPORT_ID    Report identifier.
```

Given an apropriate scan job ID or report ID, it simply retrieves and outputs the requested aggregate report. 

```
❯ poetry run qpc report aggregate --report 15 | head
{
    "ansible_hosts_all": 200,
    "ansible_hosts_in_database": 100,
    "ansible_hosts_in_jobs": 200,
    "inspect_result_status_failed": 0,
    "inspect_result_status_success": 0,
    "inspect_result_status_unknown": 601,
    "inspect_result_status_unreachable": 0,
    "instances_hypervisor": 0,
    "instances_physical": 0,
```

This command has no custom output formatting options. This command simply prints to stdout the JSON response returned by the server using qpc's existing `pretty_format` function.

```
❯ poetry run qpc report aggregate --scan-job 1701
Scan Job 1701 does not exist.
```

As discussed with Justin, we are not yet updating the man page to document this new feature. We may revisit that decision and update docs later.

### CODE REVIEW

File: qpc/tests/report/test_report_aggregate.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    scan_job_json_data = {"id": scan_job_id, "report_id": report_id}
+    aggregate_report_uri = get_aggregate_report_uri(report_id)
+    aggregate_report_json_data = {faker.slug(): faker.slug()}
+    with requests_mock.Mocker() as mocker:
+        mocker.get(scan_job_uri, status_code=200, json=scan_job_json_data)
+        mocker.get(
+            aggregate_report_uri, status_code=200, json=aggregate_report_json_data
+        )
+        args = Namespace(scan_job_id=scan_job_id, report_id=None)
+        with redirect_stdout(report_out):
```
Comment (bruno-fs) [inline qpc/tests/report/test_report_aggregate.py:75]: I didn't follow why write stdout to `report_out`, if we aren't doing anything with it. Or are we and I missed it?

Same for other similar uses of `redirect_stdout`.

TBH I nitpick a bit with this utility and would love to 🪓 it. I'm slowly replacing it with `capsys` fixture, which also captures stderr.

Comment (bruno-fs) [reply qpc/tests/report/test_report_aggregate.py:75]: Maybe for unitttest it made sense to have it just to shush qpc while testing. Nowadays with pytest this is unnecessary.

Comment (infinitewarp) [reply qpc/tests/report/test_report_aggregate.py:75]: I started by wholesale duplicating an existing test module for another command, and I chopped away a bunch of the fluff that I didn't need. If I can chop this away too, I'm happy to do it! I'll take a second look at this part.

Comment (infinitewarp) [reply qpc/tests/report/test_report_aggregate.py]: Updated and pushed. Much better now, I think. Thanks for suggesting this one!

**Review by bruno-fs — APPROVED** [review]

**Pull request merged**